tests/fast-integration/compass/client.js

```diff
@@ -251,4 +251,19 @@ class DirectorClient {
 module.exports = {
     DirectorConfig,
     DirectorClient
-};
\ No newline at end of file
+};
+
+const retry = async (fn) => {
+        try {
+            return await fn()
+        } catch (err) {
+            if (err.errors && err.errors[0].extensions.errorCode === 35) {
```

Comment:
I think we could add a comment above this line with the link to the commit in Compass (`https://github.com/kyma-incubator/compass/commit/86aa036ee48cf207230e606ee18b3cb8edbbc1e4`) so some future developers will know what the helm is this `35` :D